### PR TITLE
Make Q&A messaging for attendees clearer

### PR DESCRIPTION
**Pull request by squahtx**
After testing, one of the testers gave the following feedback:
> It isn't immediately clear what becomes a question; it seems that for a message to become a question, it has to be upvoted which wasn't clear to me in the beginning

Let's encourage attendees to thumbs up questions more clearly.

### CODE REVIEW

File: src/Scheduler.ts [hunk 1]
```diff
@@ -304,11 +304,22 @@ export class Scheduler {
             await this.scoreboard.resetScoreboard(confAud.roomId);
             if (!task.talk.prerecorded) {
                 await this.client.sendHtmlText(confTalk.roomId, `<h3>Your talk is not pre-recorded.</h3><p>You are entering the Q&A for your talk's duration now.</p>`);
-                await this.client.sendHtmlText(confAud.roomId, `<h3>${await confTalk.getName()}</h3><p><b>There is no video for this talk.</b> Ask your questions here and they'll try to answer them!</p>`);
+                await this.client.sendHtmlText(
+                    confAud.roomId,
+                    `<h3>${await confTalk.getName()}</h3>` +
+                    `<p><b>There is no video for this talk.</b> ` +
+                    `Ask your questions here and they'll try to answer them! ` +
+                    `The questions with the most 👍 votes will be answered first.</p>`,
```
Comment (turt2live) [inline src/Scheduler.ts:312]: We can't dictate human behaviour:
```suggestion
                    `The questions with the most 👍 votes are most visible to the speaker.</p>`,
```

File: src/Scheduler.ts [hunk 2]
```diff
@@ -304,11 +304,22 @@ export class Scheduler {
...
+                );
                 return;
             }
             await this.client.sendHtmlText(confTalk.roomId, `<h3>Your talk is starting shortly.</h3>`);
-            await this.client.sendHtmlText(confAud.roomId, `<h3>Up next: ${await confTalk.getName()}</h3><p>Ask your questions here for the Q&A at the end of the talk.</p>`);
+            await this.client.sendHtmlText(
+                confAud.roomId,
+                `<h3>Up next: ${await confTalk.getName()}</h3>` +
+                `<p>During the talk, you can ask questions here for the Q&A at the end. ` +
+                `The questions with the most 👍 votes will be answered first.</p>`,
```
Comment (turt2live) [inline src/Scheduler.ts:321]:
```suggestion
                `The questions with the most 👍 votes are most visible to the speaker.</p>`,
```

File: src/Scheduler.ts [hunk 3]
```diff
@@ -317,7 +328,11 @@ export class Scheduler {
                 `<p>Remember that the broadcast feed is buffered and lags many seconds behind. ` +
                 `Do not wait for it to finish, otherwise you will create a long pause!</p>`,
             );
-            await this.client.sendHtmlText(confAud.roomId, `<h3>Q&A is starting shortly</h3><p>Feel free to continue asking questions in this room for the speakers - the conversation will continue in the hallway after the Q&A.</p>`);
+            await this.client.sendHtmlText(
+                confAud.roomId,
+                `<h3>Q&A is starting shortly</h3>` +
+                `<p>Ask questions in this room for the speakers - the questions with the most 👍 votes will be answered first.</p>`,
```
Comment (turt2live) [inline src/Scheduler.ts:334]:
```suggestion
                `<p>Ask questions in this room for the speakers - the questions with the most 👍 votes are most visible to the speaker.</p>`,
```

**Review by turt2live — APPROVED** [review]
otherwise lgtm

File: src/Scheduler.ts (hunk 1 above)
Comment (squahtx) [reply src/Scheduler.ts]: The new wording is more accurate, thanks!

**Pull request merged**